pkg/graveler/graveler_test.go

```diff
@@ -2292,3 +2292,182 @@ func TestGravelerCreateCommitRecord(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestGraveler_Revert(t *testing.T) {
+	type fields struct {
```

Comment:
This name is a bit unpleasant.  Consider "deps"/"dependencies", or even "gravelerDependencies"; you could have "deps" as the name of the type, and "gravelerDeps" as the name of the field?